benchmark/_dummy-benchmarker.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
(Also, I'm fine with whatever name because this is an internal thing and we can always change it later. So if @jasnell or someone else disagrees with my suggestions, ¯\\_(ツ)_/¯.)